tutor/commands/upgrade/compose.py

```diff
@@ -165,6 +165,36 @@ def upgrade_from_quince(context: click.Context, config: Config) -> None:
     upgrade_mongodb(context, config, "6.0.14", "6.0")
     upgrade_mongodb(context, config, "7.0.7", "7.0")
 
+    if not config["RUN_MYSQL"]:
+        fmt.echo_info(
+            "You are not running MySQL (RUN_MYSQL=false). It is your "
+            "responsibility to upgrade your MySQL instance to v8.4. There is "
+            "nothing left to do to upgrade from Quince."
+        )
+        return
+
+    # Revert the MySQL image first to build data dictionary on v8.1
```

Comment:
Let's move this section to the v15 -> v16 upgrade. It's unnecessary to go through this step for users running quince or palm.

Reply:
This error happens in a particular scenario when a user upgrades their tutor instance directly from v15 -> v18. This is why I had added it in the redwood upgrade. Adding it to the v15 -> v16 would also be unnecessary, as that works just fine.

Reply:
> when a user upgrades their tutor instance directly from v15 -> v18

Don't they need to run upgrade for each version even if they installed v18 when they had v15?

Reply:
I've added it to the olive upgrade process now and added a small check to only run the code if someone is upgrading to v18 or above.